homeassistant/components/lutron/fan.py

```diff
@@ -0,0 +1,69 @@
+"""Support for Lutron fans."""
+
+from homeassistant.components.fan import DOMAIN, SUPPORT_SET_SPEED, FanEntity
+from . import DOMAIN as LUTRON_DOMAIN, LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+
+DEFAULT_ON_PERCENTAGE = 50
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None) -> None:
+    """Set up the Lutron fans."""
+    devs = []
+
+    # Add Lutron Fans
+    for (area_name, device) in hass.data[LUTRON_DEVICES][DOMAIN]:
+        dev = LutronFan(area_name, device, hass.data[LUTRON_CONTROLLER])
+        devs.append(dev)
+
+    add_entities(devs, True)
+
+
+class LutronFan(LutronDevice, FanEntity):
```

Comment:
Probably make sense to override `speed_count` to 4. Otherwise by default it will only have 3 which wont match the RRD-2ANF.